docs/Secure_Gateways_with_Istio

```diff
@@ -0,0 +1,124 @@
+# Secure Gateways with Istio
+
+## Prerequisite
+
+### Enable Istio
+
+Before installing a cluster yaml file with kubefate, please make sure you have enabled istio.
+
+```yaml
+istio:
+  enabled: false
+```
+
+
+
+### Prepare root certificate
+
+Create a root certificate and private key to sign the certificates for your services.
```

Comment:
Please help to add the content about how to setup openssl.

Reply:
Copy the contents in https://github.com/FederatedAI/KubeFATE/blob/develop-1.9.0/docs/Eggroll_with_TLS.md